ocaml/tests/test_vgpu_type.ml

```diff
@@ -174,10 +174,10 @@ module NvidiaTest = struct
         ; (* Test specific version should match *)
           (("430.19", ["430.89"; "430.19"]), true)
         ; (* Test specific version should match *)
-          (("430.19", ["430.19"; "typo_error"]), true)
-        ; (* Test specific version should match *)
```

Comment:
Comments are confusing here, they're attached to the wrong element.

Reply:
There doesn't seem to be much benefit having these comments in the first place